common/scripted_triggers/EAW_triggers.txt

```diff
@@ -1391,81 +1398,6 @@ is_a_potential_enemy_or_has_war_with_ROOT = {
 	}
 }
 
-has_dangerous_naval_war = {
```

Comment:
Why did you remove these?

Reply:
I added this a long time ago and subsequently refused as unnecessary. I haven't seen any examples of anyone using these triggers, so I decided to delete them.